src/symbol.rs

```diff
@@ -118,6 +118,12 @@ impl From<&str> for Symbol {
     }
 }
 
+impl From<String> for Symbol {
+    fn from(text: String) -> Self {
+        Symbol::owned(text)
+    }
+}
```

Comment:
🗺️ While implementing `StructBuilder` I realized you couldn't provide an owned `String` as a field name because `String` didn't implement `Into<Symbol>`.